base/services/audiosrv/pnp_list_manager.c

```diff
@@ -90,19 +85,17 @@ AppendAudioDeviceToList(PnP_AudioDevice* device)
 
     UnlockAudioDeviceList();
 
-    logmsg("Device added to list\n");
+    DPRINT("Device added to list\n");
 
     return TRUE;
 }
 
 BOOL
 CreateAudioDeviceList(DWORD max_size)
 {
-/*    printf("Initializing memory device list lock\n");*/
-
     if (!InitializeAudioDeviceListLock())
     {
-        /*printf("Failed!\n");*/
+        /*DPRINT("Failed\n");*/
```

Comment:
```suggestion
        /*DPRINT("InitializeAudioDeviceListLock() failed\n");*/
```

Reply:
not needed. it is the only failed call in here. It is even commented out. and when commented in it will contain the line number even. I just kept that dead code to highlight that this is a failure path and no good-path.